app-load-params-db.json

```diff
@@ -491,6 +491,12 @@
     "curve": ["secp256k1"],
     "path": ["45'", "1517992542'/1101353413'", "44'/60'", "44'/1'"]
   },
+  "btec": {
+    "appFlags": {"nanos": "0"},
```

Comment:
we should add at least NanoX ?

Reply:
Indeed, I re-generated the DB entry. :+1: